README.md

```diff
@@ -201,17 +201,78 @@ To create a new Breakup Simulation just follow these four steps:
 3. Create your Breakup Simulation
 4. Run and get the Result!
 
-Example:
+### Example CMake integration:
+```cmake
+cmake_minimum_required(VERSION 3.16)
+project(FOO)
 
+# ------------------------- get the breakup model -------------------------
+# Enable ExternalProject CMake module
+include(FetchContent)
+
+# OPTIONAL: disable stuff not needed when used as a lib
+set(BUILD_BREAKUP_MODEL_TESTS OFF CACHE INTERNAL "")
+set(BUILD_BREAKUP_MODEL_SIM OFF CACHE INTERNAL "")
+
+FetchContent_Declare(
+        BreakupModelfetch
+        GIT_REPOSITORY git@github.com:esa/NASA-breakup-model-cpp.git
+        # GIT_TAG master
+)
+
+# Get Breakup Model source and binary directories from CMake project
+FetchContent_MakeAvailable(BreakupModelfetch)
+
+# OPTIONAL: Disable potential warnings
+# from the library target
+target_compile_options(breakupModel_lib PRIVATE -w)
+# from included headers
+get_target_property(propval breakupModel_lib INTERFACE_INCLUDE_DIRECTORIES)
+target_include_directories(breakupModel_lib SYSTEM PUBLIC "${propval}")
+
+# -------------------- define your target and link it ---------------------
+
+add_executable(fooExe foo.cpp)
+target_compile_features(fooExe PUBLIC cxx_std_17)
+target_link_libraries(fooExe PUBLIC breakupModel_lib)
+```
+
+### Example C++ Code integration:
 ```cpp
-    //RuntimeInput via the RuntimeInputSource Object (minmial Config: minL_c = 0.05 + inputSatellites)
+#include <breakupModel/input/RuntimeInputSource.h>
+#include <breakupModel/model/Satellite.h>
+#include <breakupModel/model/SatelliteBuilder.h>
+#include <breakupModel/simulation/BreakupBuilder.h>
+
+int main() {
+    // generate satellites to collide
+    SatelliteBuilder satelliteBuilder;
+    std::vector<Satellite> satellites{
```

Comment:
So I guess the satellite type is not here as it does not impact the results? If it does, whats the default value? Maybe you can set it too here?

Reply:
The type impacts the results. From the [documentation](https://github.com/esa/NASA-breakup-model-cpp/blob/da4da134c1fb32799869b7eb15d323cd39c13da9/src/breakupModel/model/Satellite.h#L60-L67): 
> Needed for determining the right equations for breakup.
> If not given this will be always a SPACECRAFT.

I left it out since I only wanted to give a minimal example here.